src/kibana/plugins/dashboard/index.html

```diff
@@ -26,21 +28,21 @@
 
     </form>
 
-    <div class="button-group">
+    <div class="button-group" role="toolbar">
       <kbn-tooltip text="New Dashboard" placement="bottom" append-to-body="1">
-        <button ng-click="newDashboard()"><i class="fa fa-file-new-o"></i></button>
+        <button ng-click="newDashboard()" aria-label="New Dashboard"><i class="fa fa-file-new-o"></i></button>
```

Comment:
Oops. Sorry about that. I forgot the label was on the button not the icon. If the label was on the icon it would not need to be hidden as the `aria-label` text would be used in place of the psuedo content provided by font-awesome